lifecycle-operator/chart/values.yaml

```diff
@@ -0,0 +1,127 @@
+imagePullSecrets: []
+kubernetesClusterDomain: cluster.local
+schedulingGatesEnabled: false
+lifecycleManagerConfig:
+  health:
+    healthProbeBindAddress: :8081
+  leaderElection:
+    leaderElect: true
+    resourceName: 6b866dd9.keptn.sh
+  metrics:
+    bindAddress: 127.0.0.1:8080
+  webhook:
+    port: 9443
+lifecycleOperator:
```

Comment:
remove this one plz

Reply:
I would rather keep it, we still have scheduler in place, so we need to distinguish the components still...